cmd/check.go

```diff
@@ -21,10 +26,74 @@ type CheckCommandFlags struct {
 	ListPackages           bool     `json:"listPackages"`
 	Packages               []string `json:"packages"`
 	AllInstallModes        bool     `json:"allInstallModes"`
+	ExtraCRDirectory       string   `json:"extraCRDirectory"`
 }
 
 var checkflags CheckCommandFlags
 
+func handleExtraCRDirectoryFlag(capAuditor *capability.CapAuditor) {
+	logger.Debugw("scaning for extra Custom Resources", "extra CR directory", checkflags.ExtraCRDirectory)
+	extraCustomResources := map[string]interface{}{} // maps packages to a list of CR
+
+	extraCRDirectoryAbsolutePath, err := filepath.Abs(checkflags.ExtraCRDirectory)
+	if err != nil {
+		logger.Errorf("Error getting absolute path of %s: %v", checkflags.ExtraCRDirectory, err)
+	}
+
+	extraCRDirectoryContent, err := ioutil.ReadDir(extraCRDirectoryAbsolutePath)
+	if err != nil {
+		logger.Errorf("Couldn't read %s: %v", extraCRDirectoryAbsolutePath, err)
+	}
+
+	for _, packageDirectory := range extraCRDirectoryContent {
+		if packageDirectory.IsDir() {
+			packageName := packageDirectory.Name()
+			packageDirectoryPath := filepath.Join(extraCRDirectoryAbsolutePath, packageName)
+			createPackageKey := false
+			customResourceManifests := []map[string]interface{}{}
+
+			packageDirectoryContent, err := ioutil.ReadDir(packageDirectoryPath)
```

Comment:
ioutil is deprecated. os.ReadDir should be used instead.

Reply:
Fixed in eea81a145e7b39a2af3820518feb75373b791b48 (/!\ Fixup commit, don't automerge)